.golangci.yml

```diff
@@ -35,3 +44,7 @@ issues:
 linters-settings:
   gocyclo:
     min-complexity: 20
+  lll:
+    line-length: 240
```

Comment:
Trying to understand, what is the ideal value? How did we come up with 240?

Reply:
I took reference from some other project. I think we can reduce the value. Default is 120. Does it make sense to make this change in a follow up PR?